src/voloctree/voloctree_mapper.cpp

```diff
@@ -1232,32 +1232,52 @@ bool VolOctreeMapper::_recoverPartition()
     std::map<int, std::vector<Octant>> list_octant;
     std::map<int, std::vector<long>> list_id;
     std::map<int, std::vector<long>> list_globalId;
-    uint32_t idx = 0;
-    uint64_t morton = mappedPatch->getTree().getLastDescMorton(idx);
-    for (int reference_rank : toreference_rank[m_rank]) {
+
+    // Initialize idx position in mapped tree to the first mapped
+    // octant inside the overlapping region
+    std::vector<int>::iterator overlap_ref_rank_end_it = toreference_rank[m_rank].end();
+    std::vector<int>::iterator first_overlap_rank_it = overlap_ref_rank_end_it;
+    uint32_t first_overlap_ref_idx = 0;
```

Comment:
Why ref? This is not referred to mapped patch?

Reply:
Because it is an error. It should be called first_overlap_mapped_idx. I rename it.